shm/src/game.cpp

```diff
@@ -36,3 +36,11 @@ void Game::makeAction(Action action) {
                   << "\n";
     }
 }
+
+bool Game::checkWinCondition() const {
+    return money_ >= final_goal_;
+}
+
+bool Game::checkLoseCondition() const {
+    return money_ == 0;
```

Comment:
Why? I think player can loose, even when have some money, but not enough after certain amount of time - the days left. So it should always loose ;p

Reply:
You mean lose or loose?